src/Examples/UnitModels/Operations/skeleton_unit_testing.ipynb

```diff
@@ -0,0 +1,890 @@
+      "Unit : fs.pervap                                                           Time: 0.0\n",
+      "====================================================================================\n",
+      "Permeate water flow:  0.14258566520616336  mol/s\n",
+      "Permeate ethylene glycol flow:  0.000266748770992654  mol/s\n",
+      "Permeate temperature:  318.1500001701816  K\n",
+      "Permeate pressure:  1300.0  Pa\n",
+      "Retentate water flow:  0.19741533479383663  mol/s\n",
+      "Retentate ethylene glycol flow:  0.6597342512290074  mol/s\n",
+      "Retentate temperature:  318.1500001701816  K\n",
+      "Retentate pressure:  101324.9995  Pa\n"
+     ]
+    }
+   ],
+   "source": [
+    "# print results\n",
+    "m.fs.M101.report()\n",
+    "\n",
+    "print('Question for reviewers: Is there a way to add report() method with a stream table?')\n",
+    "m.fs.pervap.report()\n",
+    "\n",
+    "print('Permeate water flow: ', value(m.fs.pervap.permeate.flow_mol_phase_comp[0, \"Liq\", \"water\"]), ' mol/s')\n",
+    "print('Permeate ethylene glycol flow: ', value(m.fs.pervap.permeate.flow_mol_phase_comp[0, \"Liq\", \"ethylene_glycol\"]), ' mol/s')\n",
+    "print('Permeate temperature: ', value(m.fs.pervap.permeate.temperature[0]), ' K')\n",
+    "print('Permeate pressure: ', value(m.fs.pervap.permeate.pressure[0]), ' Pa')\n",
+    "print('Retentate water flow: ', value(m.fs.pervap.retentate.flow_mol_phase_comp[0, \"Liq\", \"water\"]), ' mol/s')\n",
+    "print('Retentate ethylene glycol flow: ', value(m.fs.pervap.retentate.flow_mol_phase_comp[0, \"Liq\", \"ethylene_glycol\"]), ' mol/s')\n",
+    "print('Retentate temperature: ', value(m.fs.pervap.retentate.temperature[0]), ' K')\n",
+    "print('Retentate pressure: ', value(m.fs.pervap.retentate.pressure[0]), ' Pa')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Inlet water mole fraction:  0.34000031999936\n",
+      "Permeate water mole fraction:  0.9981326967912821\n",
+      "Separation factor:  1037.6188149312022\n",
+      "Condensation duty:  5.812711152352085 kW\n",
+      "Duty per mole water recovered:  0.011324013423297996  kW-h / mol\n"
+     ]
+    }
+   ],
+   "source": [
+    "# separation factor for results analysis - x is inlet water fraction, y is permeate water fraction\n",
+    "m.fs.x = Expression(expr=(m.fs.pervap.inlet.flow_mol_phase_comp[0, \"Liq\", \"water\"] /\n",
+    "                           sum(m.fs.pervap.inlet.flow_mol_phase_comp[0, \"Liq\", i] for i in m.fs.pervap.comp_list)\n",
+    "                          ))\n",
+    "m.fs.y = Expression(expr=(m.fs.pervap.permeate.flow_mol_phase_comp[0, \"Liq\", \"water\"] /\n",
+    "                           sum(m.fs.pervap.permeate.flow_mol_phase_comp[0, \"Liq\", i] for i in m.fs.pervap.comp_list)\n",
+    "                          ))\n",
```

Comment:
These are also cryptic names. You could have just named them `inlet_water_frac` and `permeate_water_frac` instead of having a comment to explain your cryptic names